config/locales/liquid_tags/fr.yml

```diff
@@ -2,112 +2,112 @@
 fr:
   liquid_tags:
     asciinema_tag:
-      invalid_asciinema_id: 'Invalid Asciinema ID: %{id}'
+      invalid_asciinema_id: "ID d'Asciinema non valide : %{id}"
     blogcast_tag:
-      invalid_blogcast_id: Invalid Blogcast ID
+      invalid_blogcast_id: ID de blogcast non valide
     codepen_tag:
-      invalid_codepen_url: Invalid CodePen URL
-      invalid_options: Invalid Options
+      invalid_codepen_url: URL du CodePen non valide
+      invalid_options: Options non valides
     codesandbox_tag:
-      invalid_id: 'CodeSandbox Error: Invalid ID'
+      invalid_id: 'CodeSandbox Erreur : ID invalide'
     comment_tag:
-      invalid_comment: Invalid Comment ID or URL
+      invalid_comment: ID ou URL du commentaire non valide
     dotnet_fiddle_tag:
-      invalid_dotnetfiddle_url: Invalid DotnetFiddle URL
+      invalid_dotnetfiddle_url: URL non valide de DotnetFiddle
     gist_tag:
-      invalid_filename: Invalid Filename
-      invalid_format: 'Invalid Gist link: %{link} Links must follow this format: https://gist.github.com/username/gist_id'
-      invalid_gist_link: 'Invalid Gist link: You must provide a Gist link'
+      invalid_filename: Nom de fichier non valide
+      invalid_format: 'Lien Gist non valide : %{link} Les liens doivent respecter ce format : https://gist.github.com/username/gist_id'
+      invalid_gist_link: 'Lien Gist non valide : Vous devez fournir un lien Gist'
     git_pitch_tag:
-      invalid_gitpitch_url: Invalid GitPitch URL
+      invalid_gitpitch_url: URL GitPitch non valide
     github_tag:
       github_issue_tag:
-        comment_for: Comment for
-        commented_on: commented on
-        invalid_github_issue_pull: Invalid GitHub issue, pull request or comment link
-        posted_on: posted on
+        comment_for: Commentaire pour
+        commented_on: commenté sur
+        invalid_github_issue_pull: Lien vers une issue, une pull request ou un commentaire GitHub non valide
+        posted_on: posté sur
       github_readme_tag:
-        invalid_options: 'GitHub tag: invalid options: %{invalid} | supported options: %{valid}'
-        invalid_github_repository: Invalid GitHub repository path or URL
+        invalid_options: 'Balise GitHub : options invalides : %{invalid} | Options supportées : %{valid}'
```

Comment:
note: this "GitHub tag" refers to the markup (liquid tag) used to embed github issues/repos, and not a "Tag" from other contexts (tags on posts or listings), [_balise_](https://fr.wikipedia.org/wiki/Langage_de_balisage) looks like it makes sense.